src/OpenTelemetry.Exporter.Geneva/GenevaBaseExporter.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+#nullable enable
```

Comment:
are we in a position that we could enable nullables in the csproj and only opt-out non compliant files?

Reply:
I would keep it as is in this PR. It can be done opposite in follow ups/while cleanup while repo.